2023-07-11 01:15:33,022 - speechbrain.core - INFO - Beginning experiment!
2023-07-11 01:15:33,022 - speechbrain.core - INFO - Experiment folder: results/better_tokenizer/2005
2023-07-11 01:15:33,463 - speechbrain.utils.superpowers - DEBUG - appdirs==1.4.4
attrs==23.1.0
audioread==3.0.0
beautifulsoup4==4.12.2
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
cmake==3.26.4
decorator==5.1.1
filelock==3.12.0
fsspec==2023.6.0
gdown==4.7.1
huggingface-hub==0.15.1
HyperPyYAML==1.2.1
idna==3.4
Jinja2==3.1.2
joblib==1.2.0
jsonlines==3.1.0
kaggle==1.5.13
lazy_loader==0.2
librosa==0.10.0.post2
lit==16.0.6
llvmlite==0.40.1rc1
MarkupSafe==2.1.3
mpmath==1.3.0
msgpack==1.0.5
networkx==3.1
numba==0.57.0
numpy==1.23.5
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-cupti-cu11==11.7.101
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.2.10.91
nvidia-cusolver-cu11==11.4.0.1
nvidia-cusparse-cu11==11.7.4.91
nvidia-nccl-cu11==2.14.3
nvidia-nvtx-cu11==11.7.91
packaging==23.1
pandas==2.0.2
pooch==1.6.0
pycparser==2.21
PySocks==1.7.1
python-dateutil==2.8.2
python-slugify==8.0.1
pytz==2023.3
PyYAML==6.0
regex==2023.6.3
requests==2.31.0
ruamel.yaml==0.17.28
ruamel.yaml.clib==0.2.7
safetensors==0.3.1
scikit-learn==1.2.2
scipy==1.10.1
sentencepiece==0.1.99
six==1.16.0
sklearn==0.0.post5
soundfile==0.12.1
soupsieve==2.4.1
soxr==0.3.5
speechbrain==0.5.14
sympy==1.12
text-unidecode==1.3
threadpoolctl==3.1.0
tokenizers==0.13.3
torch==2.0.1
torchaudio==2.0.2
tqdm==4.65.0
transformers==4.30.2
triton==2.0.0
typing_extensions==4.6.3
tzdata==2023.3
urllib3==2.0.2


2023-07-11 01:15:33,467 - speechbrain.utils.superpowers - DEBUG - e2cc32f


2023-07-11 01:15:33,469 - speechbrain.core - ERROR - Exception:
Traceback (most recent call last):
  File "/home/kryo/.local/share/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8770.66/plugins/python/helpers/pydev/pydevconsole.py", line 364, in runcode
    coro = func()
           ^^^^^^
  File "<input>", line 1, in <module>
  File "/home/kryo/.local/share/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8770.66/plugins/python/helpers/pydev/_pydev_bundle/pydev_umd.py", line 198, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kryo/.local/share/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8770.66/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/kryo/Desktop/FinalProject_TrainingLab914/recipes/direct/train_trans.py", line 294, in <module>
    (train_set, valid_set, test_set, tokenizer,) = dataio_prepare(hparams)
                                                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kryo/Desktop/FinalProject_TrainingLab914/recipes/direct/dataio_prepare.py", line 12, in dataio_prepare
    train_data = sb.dataio.dataset.DynamicItemDataset.from_csv(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kryo/Desktop/FinalProject_TrainingLab914/venv/lib/python3.11/site-packages/speechbrain/dataio/dataset.py", line 365, in from_csv
    data = load_data_csv(csv_path, replacements)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kryo/Desktop/FinalProject_TrainingLab914/venv/lib/python3.11/site-packages/speechbrain/dataio/dataio.py", line 129, in load_data_csv
    with open(csv_path, newline="") as csvfile:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'results/better_tokenizer/2005/train-type=direct-sample=0.2.csv'
